### ISSUE TITLE
Screen-reader built with web technologies inc. Web Audio API

### ISSUE THREAD

**Issue by necolas** [h5bp/lazyweb-requests#101]
After watching @stuartmemo's JSConf EU talk about the Web Audio API and music - http://www.youtube.com/watch?v=PN8Eg1K9xjE - I thought about the potential to create a screen-reader using web technologies and the Web Audio API - https://twitter.com/necolas/status/262582346371432449

Not sure how practical it would be for users of AT if it cannot be used to work your computer in general (like VoiceOver et al). Maybe, if it was something like a browser extension or packaged app, running on Chrome/Mozilla OS? On the other hand, for web browsing specifically, it might have certain benefits: cross-platform, cross-browser (kinda), your settings saved to "the cloud".

Would be interesting to see what people with experience in this space think.
